renovate.json5

```diff
@@ -66,5 +54,50 @@
       "matchPackageNames": ["node-fetch", "@types/node-fetch"],
       "allowedVersions": "2.x"
     },
+    /*
+      Docs-related Renovate rules
+      These ensure that the `docs/` folder, which is its own application with
+      its own dependencies which lives inside this repository, plays by its
+      own Renovate rules.  Those rules are defined within the external (npm)
+      package called `renovate-config-apollo-docs` (defined here only by
+      the `apollo-docs` suffix).
+    */
+    {
+      "matchPaths": [
+        "docs/package.json"
+      ],
+      "extends": [
+        "apollo-docs"
+      ],
+      // We need to tell Renovate to check the branches for each major version
+      // of Apollo Server.  "Past" major versions should be added here!
+      "baseBranches": [
+        "main",
+        "version-2"
+      ],
+    },
+    // The current Apollo Gatsby theme does not support a version of Gatsby
```

Comment:
Is this still true? I'm seeing react@17 allowed by https://www.runpkg.com/?gatsby@3.14.6/package.json
Do we believe it doesn't work for other reasons?

Reply:
The docs Gatsby theme itself requires react@16.9. To be honest, with how painful the update from Gatsby v2 to v3 was, we didn't attempt to also bump the version of React that the theme uses for this go around 😬